### PR TITLE
Be explicit that ingress ipam is non-cloud only

### PR DESCRIPTION
**Pull request by marun**
This is for 3.3 and beyond only.

### CODE REVIEW

**Comment by marun** [comment]
cc: @knobunc

File: admin_guide/tcp_ingress_external_ports.adoc [hunk 1]
```diff
@@ -12,6 +12,12 @@ toc::[]
 
 == Overview
 
+Note: This feature is only supported in non-cloud deployments.  For
+cloud (GCE/AWS/OpenStack) deployments, services of type
```
Comment (knobunc) [inline admin_guide/tcp_ingress_external_ports.adoc]: Wait... do we do something special for OpenStack?

File: admin_guide/tcp_ingress_external_ports.adoc [hunk 2]
```diff
@@ -12,6 +12,12 @@ toc::[]
...
+xref:../dev_guide/getting_traffice_into_cluster.adoc#using-the-loadbalancer[LoadBalancer]
```
Comment (knobunc) [inline admin_guide/tcp_ingress_external_ports.adoc]: s/traffice/traffic//

**Review by knobunc — CHANGES_REQUESTED** [review]
Thanks Maru.

**Comment by marun** [comment]
@knobunc Would it make more sense to just move the warning on line 65?

**Comment by knobunc** [comment]
@marun I like the new wording and would leave the one at line 65 there.  My only concern is about whether we do anything special for openstack.

File: admin_guide/tcp_ingress_external_ports.adoc (hunk 1 above)
Comment (marun) [inline admin_guide/tcp_ingress_external_ports.adoc]: Yes, OpenStack supports autoprovisioning of a loadbalancer for a service of type loadbalancer just like aws and gce.

**Comment by knobunc** [comment]
LGTM. @adellape PTAL

**Review by knobunc — APPROVED** [review]
LGTM.  @adellape PTAL

File: admin_guide/tcp_ingress_external_ports.adoc (hunk 2 above)
Comment (marun) [inline admin_guide/tcp_ingress_external_ports.adoc]: Done

**Comment by ahardin-rh** [comment]
Thanks! I will make minor style edits in a follow-up PR

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/tcp_ingress_external_ports.adoc#admin-guide-unique-external-ips-ingress-traffic[Assigning Unique External IPs for Ingress Traffic]
|Added a NOTE stating that this feature is only supported in non-cloud deployments.
%

**Pull request merged**